bundle.yaml.j2

```diff
@@ -10,14 +10,12 @@ issues: https://github.com/canonical/iam-bundle/issues
 applications:
   hydra:
     charm: hydra
-    revision: 316
```

Comment:
Why remove the revision?

Reply:
for now yeah. Once everything is stabilized, we can put the revision back.